index.html

```diff
@@ -35,32 +36,81 @@
             </thead>
             <tbody>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
                 <tr>
-                    <td></td>
-                    <td></td>
+                    <td class="time"></td>
+                    <td class="description"></td>
                 </tr>
             </tbody>
```

Comment:
I understand that in the wireframe there are rows with empty spaces, which is the reason you would need to keep empty table rows
However, since we are adding values dynamically instead of modifying existing rows, just create new ones with js